glom/test/test_match.py

```diff
@@ -337,7 +336,7 @@ def none_or(sub_schema):
         'allow None or sub_schema'
         return Match(Or(None, sub_schema))
 
-    assert glom(None, none_or('abc')) == None
+    assert glom(None, none_or('abc')) == None  # noqa: E711
```

Comment:
```suggestion
    assert glom(None, none_or('abc')) is None
```

Reply:
basically, the ones related to matching have to do `==` bc you can't override `is`. But this one, pretty sure could be an `is`. :)